arch/arm/boot/dts/overlays/Makefile

```diff
@@ -219,6 +220,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	vc4-kms-dsi-lt070me05000.dtbo \
 	vc4-kms-dsi-lt070me05000-v2.dtbo \
 	vc4-kms-kippah-7inch.dtbo \
+	vc4-kms-at056tn53v1.dtbo \
```

Comment:
Maintain the alphabetical order, please.

Reply:
Will fix this.

Reply:
Ahem.